otp4gb/config.py

```diff
@@ -50,13 +53,14 @@ class ProcessConfig(caf.toolkit.BaseConfig):
     centroids: str
     destination_centroids: Optional[str] = None
     iterinary_aggregation_method: cost.AggregationMethod = cost.AggregationMethod.MEAN
-    max_walk_distance: int = 2500
-    number_of_threads: pydantic.conint(ge=0, le=10) = 0
+    max_walk_distance: int = pydantic.Field(2500, ge=0)
+    number_of_threads: int = pydantic.Field(0, ge=0, le=10)
```

Comment:
Why are threads capped to 10?

Reply:
There were some issues when sending too many requests to OTP at once, but it probably will vary depending on the machine so hardcoded cap of 10 is probably a bit small